rpmbuild/copr-rpmbuild.spec

```diff
@@ -282,7 +291,7 @@ install -p -m 755 copr-update-builder %buildroot%_bindir
 )
 
 install -p -m 755 bin/copr-distgit-client %buildroot%_bindir
-%if 0%{?fedora} || 0%{?rhel} > 7
+%if 0%{?fedora} || 0%{?rhel} > 7 || 0%{?openEuler}
```

Comment:
I think we can bury the EL7 support here.  Can you please drop the condition entirely?

Reply:
Done